man/ocitools-generate.1.md

```diff
@@ -84,6 +84,45 @@ inside of the container.
   The special *PATH* `host` removes any existing IPC namespace from the
   configuration.
 
+**--linux-cpu-shares**=CPUSHARES
+  Specifies a relative share of CPU time available to the tasks in a cgroup.
+
+**--linux-cpu-period**=CPUPERIOD
+  Specifies a period of time in microseconds for how regularly a cgroup's access to CPU resources should be reallocated (CFS scheduler only).
```

Comment:
“CFS only” seems like we want a `--host-specific` guard to error out if someone tries to set it with `--host-specific` when they aren't using a CFS scheduler.  But after reading over [sched(7)][1] and [sched_setscheduler(2)][2], I'm not quite sure how to do that.  It looks like you check for a `SCHED_OTHER` `policy`, but `sched_getscheduler` is per-thread.  Is the scheduler policy really set on a per-thread basis?  Is a host-specific check just supposed to look for internal consistency (e.g. the config doesn't set both a CFS-only and a realtime-only parameter)?

[1]: http://man7.org/linux/man-pages/man7/sched.7.html
[2]: http://man7.org/linux/man-pages/man2/sched_setscheduler.7.html